### ISSUE TITLE
"Grasp" conflated with GRASP the gene

### ISSUE THREAD

**Issue by amykglen** [RTXteam/RTX#1545]
in KG2.6.7c:
```
{
  "iri": "http://purl.obolibrary.org/obo/OMIM_612027",
  "expanded_categories": [
    "biolink:Gene",
    "biolink:Protein",
    "biolink:GenomicEntity",
    "biolink:MolecularEntity",
    "biolink:BiologicalEntity",
    "biolink:NamedThing"
  ],
  "name": "GRASP",
  "description": "The ability to hold or grasp.; Force exerted when gripping or grasping.; UMLS Semantic Type: UMLS_STY:T040",
  "equivalent_curies": [
    "UMLS:C0220843",
    "OMIM:612027"
  ],
  "id": "OMIM:612027",
  "category": "biolink:Gene",
  "all_names": [
    "GRASP",
    "grasp"
  ],
  "all_categories": [
    "biolink:PhysiologicalProcess",
    "biolink:Gene"
  ]
}
```
(this conflation is also present in KG2.5.2c, although worse)

I think probably an exception will need to be added for this one?

**Comment by amykglen** [comment]
this one is still present in KG2.6.7.1c: https://arax.ncats.io/?term=OMIM:612027

**Comment by edeutsch** [comment]
I think the easiest interim solution is to put in an exception to rename UMLS:C0220843 to "grasp (action)"

**Comment by finnagin** [comment]
@edeutsch @amykglen is this issue okay to close?

**Comment by amykglen** [comment]
looks like this has been fixed with the new synonymizer (#2003):

Cluster for OMIM:612027 (NCBIGene:160622) has 9 nodes:

| id                      | category   | name                                | in_SRI   | in_KG2pre   | is_cluster_rep   |
|:------------------------|:-----------|:------------------------------------|:---------|:------------|:-----------------|
| ENSEMBL:ENSG00000161835 | Gene       | TAMALIN                             | X        | X           |                  |
| HGNC:18707              | Gene       | TAMALIN                             | X        | X           |                  |
| NCBIGene:160622         | Gene       | TAMALIN                             | X        | X           | X                |
| NCIT:C115266            | Gene       | TAMALIN Gene                        |          | X           |                  |
| OMIM:612027             | Gene       | TAMALIN                             | X        | X           |                  |
| PR:Q7Z6J2               | Protein    | protein TAMALIN (human)             | X        | X           |                  |
| UMLS:C1425726           | Gene       | TAMALIN gene                        | X        | X           |                  |
| UMLS:C1433105           | Protein    | GRASP protein, human                | X        | X           |                  |
| UniProtKB:Q7Z6J2        | Protein    | GRASP_HUMAN Protein TAMALIN (sprot) | X        | X           |                  |

Cluster for UMLS:C0220843 has 1 nodes:

| id            | category             | name   | in_SRI   | in_KG2pre   | is_cluster_rep   |
|:--------------|:---------------------|:-------|:---------|:------------|:-----------------|
| UMLS:C0220843 | PhysiologicalProcess | grasp  | X        | X           | X                |

**Comment by amykglen** [comment]
confirmed fixed on our dev instances:

https://arax.ncats.io/devLM/?term=NCBIGene:160622
https://arax.ncats.io/devLM/?term=UMLS:C0220843
